package.json

```diff
@@ -2269,7 +2265,7 @@
         "del": "^3.0.0",
         "enzyme": "^3.7.0",
         "enzyme-adapter-react-16": "^1.6.0",
-        "eslint": "^7.17.0",
+        "eslint": "^8.8.0",
```

Comment:
Looks like our Lint step might have issues with this?

Reply:
Yeah it looks like all of the parsers in use aren't supported yet. I moved this back.